config/routes.rb

```diff
@@ -5,8 +5,11 @@
     resources :seances, except: [:show, :index] do
       resources :reservations
     end
-  end     
-resources :seances, :reservations
+  end 
+  resources :seances
+  resources :reservations
+  get '/find_reservations_by_user', to: "reservations#find_reservations_by_user"
```

Comment:
You can consider [collection routes ](https://guides.rubyonrails.org/routing.html#adding-collection-routes)for this purpose :wink:

Reply:
Closed this routes in collection, thanks :D